PKGBUILD

```diff
@@ -0,0 +1,75 @@
+# Maintainer: hodasemi <michaelh.95 at t-online dot de>
+_pkgbase=breezy-desktop
+pkgname="${_pkgbase}"
+pkgver=0.1
+pkgrel=1
+pkgdesc="Breezy desktop - XR desktop"
+arch=('x86_64')
+url="https://github.com/wheaney/breezy-desktop"
+license=('GPL-3.0')
+makedepends=('ninja' 'meson')
+depends=('python' 'python-pydbus' 'gnome-shell')
+conflicts=("${_pkgbase}")
+source=("git+${url}")
+md5sums=(SKIP)
+
+_uuid="breezydesktop@xronlinux.com"
+
+build() {
+    cd ${_pkgbase}
+
+    # init submpdules
+    git submodule update --init --recursive
+
+    # build binaries
+    cd ui
+    meson setup build
+    cd build
+    meson compile
+
+    # prepare extension
+    cd ../..
+    unlink gnome/src/schemas/com.xronlinux.BreezyDesktop.gschema.xml
+    cp ui/data/com.xronlinux.BreezyDesktop.gschema.xml gnome/src/schemas/
+    glib-compile-schemas --targetdir="gnome/src/schemas" "gnome/src/schemas"
+
+    unlink gnome/src/textures/custom_banner.png
+    cp vulkan/custom_banner.png gnome/src/textures/
+
+    unlink gnome/src/textures/calibrating.png
+    cp modules/sombrero/calibrating.png gnome/src/textures/
+
+    # build xr driver
+    cd modules/XRLinuxDriver
+    mkdir build/
+    cd build
+    cmake ..
+    make
+}
+
+package() {
+    # copy gnome extension
+    install -Dm755 ${_pkgbase}/ui/data/com.xronlinux.BreezyDesktop.gschema.xml "${pkgdir}"/usr/share/glib-2.0/schemas/com.xronlinux.BreezyDesktop.gschema.xml
+
+    install -d "${pkgdir}/usr/share/gnome-shell/extensions/${_uuid}/"
+    cp -r ${_pkgbase}/gnome/src/* "${pkgdir}/usr/share/gnome-shell/extensions/${_uuid}/"
+    unlink "${pkgdir}/usr/share/gnome-shell/extensions/${_uuid}/IMUAdjust.frag"
```

Comment:
Is there no way to use the `gnome-extensions` command to install the extension? I don't like the idea of manually installing it, but if we have to then I guess that's fine.

Reply:
I'm also a little skeptical that this will work with the glib compiled schema, as I think `gnome-extensions` might do more than just copy files. Before I got this install completely working, I was testing (and having others test) the extension via manual copies like this, and it required special setup of the compiled schema file. I wonder if this would work if you tried this install process on a fresh OS install (you may already have the compiled schemas registered if you've done it using my installer), I wouldn't be shocked if it doesn't.

Reply:
I oriented myself on the other packages in the arch repo ([e.g.](https://gitlab.archlinux.org/archlinux/packaging/packages/gnome-shell-extension-appindicator/-/blob/main/PKGBUILD?ref_type=heads)). 
`meson install` is basically just a copy command.

As you can see in the compile step I create everything there.

Reply:
If it works for AppIndicator then it would seem like it must work for us, but when I used glib-compile-schemas and kept the schemas/ dir in the extenion dir (as you're doing here), it wouldn't properly work for me. I can't remember exactly, but it may be that my Gtk Breezy Desktop UI app complicated things because it's trying to sync settings with the extension, and it runs from a Flatpak where the syncing of dconf with the host machine was pretty complicated. So I think you may find if you install it like this, then the Breezy Desktop app settings won't sync with the extension.

Reply:
I'm currently working on the XRLinuxDriver, that the base runs as it should and then I would take a look what it takes to get breezy desktop running via pkg file.

Reply:
I have made some research regarding the `gnome-extensions` tool:

The tool only support installing extensions as user extensions and not on a system level. Thus it is not of much use in this case.

Reply:
Taken from the man page:

```
install [OPTION...] PACK
           Installs an extension from the bundle PACK.

           The command unpacks the extension files and moves them to the
           expected location in the user’s $HOME, so that it will be loaded in
           the next session.

           It is mainly intended for testing, not as a replacement for the
           extension website. As extensions have privileged access to the
           user’s session, it is advised to never load extensions from
           untrusted sources without carefully reviewing their content.

           Options
               --force
                   Override an existing extension

```

Reply:
I'm only installing as a user extension, so that's fine.

Reply:
Right. 

But: `gnome-extensions` only mimics what https://extensions.gnome.org/ does. It install a `gnome-extension` on a `user` level. What a package does is, it installs a `gnome-extension` on a `system` level accessible for every user with 1 installation.

Reply:
> If it works for AppIndicator then it would seem like it must work for us

The AppIndicator pkg file also does not use the tool: https://gitlab.archlinux.org/archlinux/packaging/packages/gnome-shell-extension-appindicator/-/blob/main/PKGBUILD?ref_type=heads